ci-operator/step-registry/openshift-qe/network-test/openshift-qe-network-test-workflow.yaml

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
should we take out this new folder and separate it into a new PR? Looks like [this code](https://github.com/openshift/release/tree/master/ci-operator/step-registry/openshift-qe/network-perf) is actually already merged. Think you just need to rebase

Reply:
this is wrong file, I have remove those files and rebase the code, thanks!